python/src/cloudstorage/.settings/de.loskutov.FileSync.prefs

```diff
@@ -0,0 +1,7 @@
+WARNING=DO NOT MODIFY THIS FILE IF YOU DON'T UNDERSTAND
```

Comment:
It was in the git ignore. I had to manually delete it before the update would remove it. Its gone now.